pkgs/development/libraries/spice-gtk/default.nix

```diff
@@ -94,7 +100,10 @@ stdenv.mkDerivation rec {
     python3
     python3.pkgs.pyparsing
     python3.pkgs.six
+    wayland-scanner
```

Comment:
This should be guarded under `stdenv.isLinux`.

Reply:
can you open a PR?